server/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "server.js",
   "dependencies": {
     "body-parser": "^1.18.3",
-    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#v6.0.0",
+    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#td-add-sound-to-push-notifications",
```

Comment:
Reminder to self to update this once https://github.com/bravetechnologycoop/brave-alert-lib/pull/30 is merged.